fbw-common/src/systems/instruments/src/EFB/AircraftContext.ts

```diff
@@ -10,12 +10,55 @@ interface PerformanceCalculators {
   landing: LandingPerformanceCalculator | null;
 }
 
+interface PushbackPage {
+  turnIndicatorTuningDefault: number;
+}
 interface SettingsPages {
   autoCalloutsPage: React.ComponentType<any>;
+  audio: AudioOptions;
+  pinProgram: PinProgramOptions;
+  realism: RealismOptions;
+  sim: SimOptions;
+  throttle: ThrottleOptions;
+}
+
+interface AudioOptions {
+  announcements: boolean;
+  boardingMusic: boolean;
+  masterVolume: boolean;
+  windVolume: boolean;
+  engineVolume: boolean;
+  paxAmbience: boolean;
+  ptuCockpit: boolean;
+}
+interface PinProgramOptions {
+  latLonExtend: boolean;
+  paxSign: boolean;
+  rmpVhfSpacing: boolean;
+  satcom: boolean;
+}
+
+interface RealismOptions {
+  mcduKeyboard: boolean;
+  pauseOnTod: boolean;
```

Comment:
Doesn't need to be configurable as both planes have it (and all planes we might do in future also would).

Reply:
True but may be useful to use this to be able to quickly disable certain functionality/settings if a sim update breaks something. Also noticed in FS24 tech alpha that there is a new system for notifications (which pause on TOD uses). So may need to disable this in any initial FS24 release.